views/shop/refundtopoints.tt.text

```diff
@@ -9,6 +9,8 @@
 
 .complete=Your exchange has been completed.
 
+.nextrefund=You won't be able to use this tool again until after [[date]].
```

Comment:
Suggest wording change: "You'll be able to [apply refunds?] again after [[date]]"